osu.Game.Rulesets.Osu/Mods/OsuModAutopilot.cs

```diff
@@ -62,7 +62,8 @@ public void ApplyToDrawableRuleset(DrawableRuleset<OsuHitObject> drawableRuleset
             inputManager.AllowUserCursorMovement = false;
 
             // Generate the replay frames the cursor should follow
-            replayFrames = new OsuAutoGenerator(drawableRuleset.Beatmap).Generate().Frames.Cast<OsuReplayFrame>().ToList();
+            var mods = (IReadOnlyList<Mod>)drawableRuleset.Dependencies.Get(typeof(IReadOnlyList<Mod>));
```

Comment:
wondering if we should expose this on `DrawableRuleset` instead... @smoogipoo thoughts?

Reply:
Agree with exposing from `DrawableRuleset`.